src/data/launch.rs

```diff
@@ -100,6 +100,7 @@ pub struct Slice {
 /// assert!(invalid.is_err());
 /// ```
 #[derive(Serialize, Debug)]
+#[derive(PartialEq)]
```

Comment:
Should `Eq` be added too?
https://doc.rust-lang.org/std/cmp/trait.Eq.html

Also, should the two `derive` usages be combined?

Reply:
I think `Eq` is what we want here, based on definitions in the linked docs. Since `Eq` always requires `PartialEq`, we have to implement them both.

> Also, should the two derive usages be combined?
`rustfmt` says so.

Reply:
If I'm understanding correctly. The difference is that `Eq` enforces `reflexive` while `PartialEq` does not. Using the example from PartialEq `Nan` cannot be reflexive(ly) equal. 

As a side note, I'm not sure why `PartialEq` implements default `==` but `Eq` doesn't.

```
error[E0277]: can't compare `launch::ProcessType` with `launch::ProcessType`
   --> src/data/launch.rs:102:28
    |
102 | #[derive(Serialize, Debug, Eq)]
    |                            ^^ no implementation for `launch::ProcessType == launch::ProcessType`
    |
   ::: /Users/rschneeman/.rustup/toolchains/stable-x86_64-apple-darwin/lib/rustlib/src/rust/library/core/src/cmp.rs:268:15
    |
268 | pub trait Eq: PartialEq<Self> {
    |               --------------- required by this bound in `Eq`
    |
    = help: the trait `PartialEq` is not implemented for `launch::ProcessType`
    = note: this error originates in the derive macro `Eq` (in Nightly builds, run with -Z macro-backtrace for more info)
```

Another way: Why isn't `Eq` derive a superset of `PartialEq` ?

Reply:
PR is updated